features/support/object_model_pages/admin/homepage/admin_homepage.rb

```diff
@@ -106,4 +106,12 @@ def self.help_link
   def self.log_out_link
     '.interaction-click-control-logout'
   end
+
+  def self.medicaid_banner_text
+    '[data-cuke="medicaid-banner-text"]'
+  end
+
+  def self.medicaid_button_text
+    '[data-cuke="medicaid-button-text"]'
```

Comment:
where does this attribute appear in the html?

Reply:
this is no longer needed ,this will be removed on next commit.